### ISSUE TITLE
[a11y] The "password rules" in account creation cannot be dismissed

### ISSUE THREAD

**Issue by liuche** [mozilla/fxa#6797]
## Description

In Fenix, QA filed a bug about a11y in the FxA creation page.

## Steps to reproduce
1. Turn voice assistant or talkback on your device.
2. Go to Settings - Turn on sync and create a new account.

### Expected behavior
The user can easily create an account.

### Actual behavior
The user is having difficulties confirming the password; the password tips cannot be dismissed, blocking the confirmation of the password.
Workaround: complete the checkboxes below the password area, then return to the password confirmation.

### Device information
* Android device: Samsung Galaxy Tab S6 (Android 10)
* Fenix version: 10/19 Nightly, RC 82.1.0-rc.2, Beta 82.0.0-beta.6

<img src="https://user-images.githubusercontent.com/40694158/96467772-f819bb00-1233-11eb-9e0d-9b0f218e6829.png" width="300" height="550" />
 

Fenix issue: https://github.com/mozilla-mobile/fenix/issues/16028
